README.md

```diff
@@ -57,6 +57,52 @@ export class OtherModule {
 }
 ```
 
+### Including only what you need
+For production scenarios, you will likely only want to include the parts of ng-bootstrap that you are using in your application.
+To do so, you can import only the specific modules that you need. For example:
+
+ ```js
+ import { NgbModalModule } from '@ng-bootstrap/ng-bootstrap/modal/modal.module';
```

Comment:
This is maybe more important for lazy loading - it might be worth noting for that specific scenario.